snippets/sdk/python/deployments/create_or_update.py

```diff
@@ -0,0 +1,90 @@
+#! /usr/local/env python3
+
+import os
+
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.nginx import NginxManagementClient
+from azure.mgmt.network import NetworkManagementClient
+
+import prerequisites
+
+
+def main():
+    SUBSCRIPTION_ID = os.environ.get("AZURE_SUBSCRIPTION_ID", None)
+    GROUP_NAME = "myResourceGroup"
+    DEPLOYMENT_NAME = "myDeployment"
+
+    # Create clients
+    # For other authentication approaches, please see: https://pypi.org/project/azure-identity/
+    nginx_client = NginxManagementClient(
+        credential=DefaultAzureCredential(), subscription_id=SUBSCRIPTION_ID
+    )
+    # Create prerequisite resources
+    subnet, public_ip, identity = prerequisites.create()
+
+    # - private IP configuration example
+    network_profile_private = {
```

Comment:
unused var?

Reply:
This is to provide folks an example of the structure needed for a private IP. I can move this to a different snippet though like we do for arm-templates.